packages/server/graphql/mutations/helpers/notifications/MattermostNotifier.ts

```diff
@@ -87,6 +90,71 @@ const makeEndMeetingButtons = (meeting: Meeting) => {
 
 type MattermostNotificationAuth = IntegrationProviderMattermost & {userId: string}
 
+const makeTeamPromptStartMeetingNotification = (
+  team: Team,
+  meeting: Meeting,
+  meetingUrl: string
+) => {
+  return [
+    makeFieldsAttachment(
+      [
+        {
+          short: true,
+          title: 'Team',
+          value: team.name
+        },
+        {
+          short: false,
+          value: makeHackedFieldButtonValue({label: 'Submit Response', link: meetingUrl})
+        }
+      ],
+      {
+        fallback: `Async Standup started, submit response: ${meetingUrl}`,
+        title: `${meeting.name} is open 💬 `,
+        title_link: meetingUrl
+      }
+    )
+  ]
+}
+
+const makeGenericStartMeetingNotification = (team: Team, meeting: Meeting, meetingUrl: string) => {
+  return [
+    makeFieldsAttachment(
+      [
+        {
+          short: true,
+          title: 'Team',
+          value: team.name
+        },
+        {
+          short: true,
+          title: 'Meeting',
+          value: meeting.name
+        },
+        {
+          short: false,
+          value: makeHackedFieldButtonValue({label: 'Join meeting', link: meetingUrl})
+        }
+      ],
+      {
+        fallback: `Meeting started, join: ${meetingUrl}`,
+        title: 'Meeting started 👋',
+        title_link: meetingUrl
+      }
+    )
+  ]
+}
+
+const makeStartMeetingNotificationLookup: Record<
+  MeetingTypeEnum,
+  (team: Team, meeting: Meeting, meetingUrl: string) => FieldAttachment[]
```

Comment:
+1

personally, i don't like exporting anything I don't have to.
you could replace `FieldAttachment` with `ReturnType<typeof makeGenericStartMeetingNotification>` and then if  `makeGenericStartMeetingNotification` ever changes its signature you don't have to update anything. you could also stop exporting `FieldAttachment`. up to you

Reply:
fixed! except for doing `ReturnType<typeof makeGenericStartMeetingNotification>` I did `ReturnType<typeof makeFieldsAttachment>[]` to make `makeStartMeetingNotificationLookup` compatible with both `makeTeamPromptStartMeetingNotification` and `makeGenericStartMeetingNotification`